### ISSUE TITLE
use `_default` instead of `` as dest in complete_zsh

### ISSUE THREAD

**Issue by tshu-w** [iterative/shtab#65]
Hi, it will be much better if we use `_default` instead of `` as dest in `complete_zsh` https://github.com/iterative/shtab/blob/master/shtab/__init__.py#L476 and https://github.com/iterative/shtab/blob/master/shtab/__init__.py#L486. User can get basic default competition without setting.

**Comment by casperdcl** [comment]
Hi @tshu-w - not sure I follow but would be happy to accept a PR :)

**Comment by casperdcl** [comment]
Makes sense; interface could be `shtab.add_argument_to(..., default_complete: dict[str, str])`

**Comment by tshu-w** [comment]
@casperdcl Yes, I'm looking forward to this enhancement.

**Comment by casperdcl** [comment]
actually... is this incompatible with #46?

**Comment by tshu-w** [comment]
Not really. What I want is a better default completion target, #46 is to autofill the `%(default)s` in the help message.

**Comment by casperdcl** [comment]
Does #83 work for you?

**Comment by casperdcl** [comment]
Having thought about this; this is deliberately disabled by default.
You can explicitly have completions by doing:

```py
parser.add_argument('positional').complete = shtab.FILE
```

Now mentioned in https://tqdm.github.io/shtab/#faqs

**Comment by mauvilsa** [comment]
@tshu-w this issue was created long ago, and jsonargparse now has native shtab support, so the example
```
run fit --model configs/model/default.yaml --data
```
should already work. See the [docs](https://jsonargparse.readthedocs.io/en/v4.50.0/#shtab). If it doesn't work for some reason, that is something that could be fixed on jsonargparse side.
